README.md

```diff
@@ -24,29 +24,33 @@ Memory requirements for the VM bootstrapped with Vagrant: 2GB (i.e. you need a m
 
 ## Pre-requisites: URL mapping
 
-You are encouraged to edit the `/etc/hosts` file (on windows: `C:\Windows\system32\drivers\etc\hosts`) to add the entry:
+You are encouraged to edit the `/etc/hosts` file on the HOST machine (on windows: `C:\Windows\system32\drivers\etc\hosts`) to add the entry:
 
-Pure docker case: `127.0.0.1 appserver.test my.appserver.test traefik.appserver.test portainer.appserver.test kibana.appserver.test`
+Vagrant case: `192.168.168.168 appserver.test my.appserver.test traefik.appserver.test portainer.appserver.test`
```

Comment:
I left it out intentionally 'as an exercise' for the person to do it themselves, as you see below pgadmin is 'the' example i picked to add a new host, domain, and service. What do you think?

Reply:
I must say it's not the place I'm looking for exercises. Not in the setup guide. Others can weigh on this too.

Reply:
I would say maybe even remove traefik, portainer and pgadmin since they are also mentioned in the table for subdomains (line 163 and further). Since we would not really use these at the start of development afaik